manifests/profiles/demo.yaml

```diff
@@ -12,6 +12,11 @@ spec:
       skywalking:
         service: tracing.istio-system.svc.cluster.local
         port: 11800
+    - name: otel
+      opentelemetry:
+        port: 4317
+        service: opentelemetry-collector.istio-system.svc.cluster.local
+    enableTracing: true
```

Comment:
Should't this only be about configuring the provider ? We have an API to configure tracing, why would it not be enabled by default ( with 0% sampling ),
and opt-on at workload level via the API ?

Reply:
+1 this seems weird

Reply:
+2 on enableTracing